pkg/networkfs/resolvconf/resolvconf.go

```diff
@@ -12,9 +12,16 @@ import (
 )
 
 var (
-	defaultDns      = []string{"8.8.8.8", "8.8.4.4"}
-	localHostRegexp = regexp.MustCompile(`(?m)^nameserver 127[^\n]+\n*`)
-	nsRegexp        = regexp.MustCompile(`^\s*nameserver\s*(([0-9]+\.){3}([0-9]+))\s*$`)
+	// Note: the default IPv4 & IPv6 resolvers are set to Google's Public DNS
+	defaultIPv4Dns = []string{"8.8.8.8", "8.8.4.4"}
+	defaultIPv6Dns = []string{"2001:4860:4860::8888", "2001:4860:4860::8844"}
+	ipv4NumBlock   = `(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)`
+	ipv4Address    = `(` + ipv4NumBlock + `\.){3}` + ipv4NumBlock
+	ipv6Address    = `([0-9A-Fa-f]{0,4}:){2,7}([0-9A-Fa-f]{1,4})`
```

Comment:
Besides, link-local IPv6 addresses like `fe80::a:b:c:d%eth0` that work on the host will not work from within the container.
But I think the person who's using link-local addresses for DNS should know what he's doing (wrong).